src/index.html

```diff
@@ -9,6 +9,90 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="container">
+        <div class="wrapper">
```

Comment:
too many wrappers, you can make this structure and that will be enough

```suggestion
      <div class="container">
        <header class="header">
```